cmd/bnscli/clitests/create_multisig_contract.test

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+bnscli multisig -activation 4 -admin 8 \
```

Comment:
This is a very nice and clear approach.
I like the multiple `with-mutlisig-participant` instead of some csv file :)